.github/BUG_REPORT.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Bug Report
+about: Report a bug encountered while using Kueue
+labels: kind/bug
+
+---
+
+<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!
+
+If the matter is security related, please disclose it privately via https://kubernetes.io/security/
+-->
+
+
+**What happened**:
+
+**What you expected to happen**:
+
+**How to reproduce it (as minimally and precisely as possible)**:
+
+**Anything else we need to know?**:
+
+**Environment**:
+- Kubernetes version (use `kubectl version`):
```

Comment:
kueue version as well (although we don't have one yet), so it could be commit hash too

Reply:
done